src/demo/DemoTestLoanService.java

```diff
@@ -2452,10 +2452,10 @@ public static void testAddAndRemoveSetllementAccounts() throws MambuApiException
 		System.out.println("Obtaining the product type of the loan account");
 		LoanProduct loanProduct = loanService.getLoanProduct(productTypeKey);
 
-		if (!loanProduct.isAccountLinkingEnabled()) {
+		if (!loanProduct.isAccountLinkingEnabled() || loanProduct.isAutoCreateLinkedAccounts()) {
```

Comment:
From the business point of view currently this case doesn't exist in Mambu ( at least for the moment). Nevertheless the case you describe loanProduct.isAccountLinkingEnabled()= false and isAutoCreateLinkedAccounts() =true will be short-circuited and the condition loanProduct.isAutoCreateLinkedAccounts() will not be executed, at all. All I need, is to check with all this if is to check that the saving account will not be created in case the loan is not linkable and also the saving account will not be created in case  loanProduct.isAutoCreateLinkedAccounts()== true,  this means a saving account will be created automatically once with the loan account and the demo test doesn't have to create it.